src/services/dashboards/all-dashboards/AllDashboardsPage.vue

```diff
@@ -1,11 +1,174 @@
 <template>
     <div class="all-dashboards-page">
-        DASHBOARDs PAGE
+        <p-page-title :title="'Dashboards'"
+                      use-total-count
+        >
+            <template #extra>
+                <p-button icon-left="ic_plus"
+                          @click="handleCreateDashboard"
+                >
+                    <!--song lang-->
+                    Create
+                </p-button>
+            </template>
+        </p-page-title>
+        <p-divider class="dashboards-divider" />
+        <all-dashboards-select-filter :viewer-status.sync="viewersStatus"
+                                      :scope-status.sync="scopeStatus"
+        />
+        <p-search :value="keyword" />
+        <div class="dashboard-list-wrapper">
+            <dashboard-board-list class="dashboard-list"
+                                  :field-title="'Entire Workspace'"
+                                  :dashboard-list="workspaceDashboardList"
+            />
+            <dashboard-board-list class="dashboard-list"
+                                  :field-title="'Single Project'"
+                                  :dashboard-list="projectDashboardList"
+            />
+        </div>
     </div>
 </template>
 
 <script lang="ts">
+import {
+    computed,
+    reactive, toRefs,
+} from 'vue';
+
+import {
+    PPageTitle, PDivider, PButton, PSearch,
+} from '@spaceone/design-system';
+
+import { SpaceRouter } from '@/router';
+import { i18n } from '@/translations';
+
+import AllDashboardsSelectFilter from '@/services/dashboards/all-dashboards/modules/AllDashboardsSelectFilter.vue';
+import DashboardBoardList from '@/services/dashboards/all-dashboards/modules/DashboardBoardList.vue';
+import { DASHBOARDS_ROUTE } from '@/services/dashboards/route-config';
+
 export default {
     name: 'AllDashboardsPage',
+    components: {
+        DashboardBoardList,
+        PSearch,
+        AllDashboardsSelectFilter,
+        PButton,
+        PPageTitle,
+        PDivider,
+    },
+    setup() {
+        const state = reactive({
+            keyword: '',
+            viewersStatus: { label: i18n.t('All'), name: 'ALL' },
+            scopeStatus: { label: i18n.t('All'), name: 'ALL' },
```

Comment:
issue: It must recalculated when the locale is changed. To make work as expected, each of them must be computed ref.

Reply:
It may refactor soon..!